### ISSUE TITLE
Boostrap MNIST tutorial

### ISSUE THREAD

**Issue by uslumt** [lava-nc/lava-dl#69]
Objective of issue:

**Lava DL version:**

- [ ] **0.3.0** (feature release)
- [ ] **0.2.1** (bug fixes)
- [x] **0.2.0** (current version)
- [ ] **0.1.2**

**Lava version:**

- [ ] **0.4.0** (feature release)
- [ ] **0.3.1** (bug fixes)
- [x] **0.3.0** (current version)
- [ ] **0.2.0**
- [ ] **0.1.2**

**I'm submitting a ...** 

- [ ] bug report
- [ ] feature request
- [ ] documentation request

**Current behavior:**

- 

**Expected behavior:**

- 

**Steps to reproduce:**

- 

**Related code:**

```
insert short code snippets here
```

**Other information:**

```
insert the output from lava debug here
```

**Comment by bamsumit** [comment]
@uslumt Thanks for raising the issue.

Can you try changing line 64 in `tutorials/lava/lib/dl/bootstrap/mnist/mnist.py` to
`return x, torch.FloatTensor(count).reshape((1, -1)).to(x.device)`

and let us know if that works for you. If it works, we will issue a fix.

**Comment by uslumt** [comment]
Hello,
return x, torch.FloatTensor(count).reshape((1, -1)).to(x.device) and return x, torch.FloatTensor(count).reshape((1, -1)).to(device) works fine but as I mentioned in the issue get_device function returns negative index.
Thanks for your interest, I hope it'll be helpful for other developers.

**Comment by bamsumit** [comment]
Exactly. `x.device` does not return an integer like `get_device()`, so it should work for both cases.
